ArduCopter/config.h

```diff
@@ -196,6 +196,8 @@
 // flip - fly vehicle in flip in pitch and roll direction mode
 #ifndef MODE_FLIP_ENABLED
 # define MODE_FLIP_ENABLED ENABLED
+# define FLIP_MODE FLIP_RIGHT
+# define FLIP_THROTTLE 0.0f
```

Comment:
I don't think its worth these defines, i fact I don't think your even using them?

Reply:
You are correct, I misread the wiki article. I will remove them in the next commit.